sorc/link_workflow.sh

```diff
@@ -278,36 +286,39 @@ done
 #--link source code directories
 #------------------------------
 
+    [[ -d ncdiag.fd ]] && rm -rf ncdiag.fd
```

Comment:
**Two comments.**
1) We are still in `exec/` when this section is executed.  We should move line 298, namely, 

    `cd ${pwd}/../sorc   ||   exit 8`

    to just before line 289.   Otherwise the ndiag, global_gsi and global_enkf source directories are created in the wrong location.

2)  I think operational Implementation standards require the source code and executable name to be consistent.  For example, if the executable is `gsi.x`, the source code directory should be `gsi.fd`.   If this is true, we need to change the source code directory names in this section to be consistent with the executable name.

Reply:
Thanks @RussTreadon-NOAA for catching the linking of the directories.
I have retained the names of the executables to be consistent with the source code.  `ncdiag.fd` will become moot soon.

Reply:
It dawned on me that we already have `gsi.fd` from `checkout.sh`.   Thus, the following section of `link_workflow.sh` is not needed
```
    [[ -d gsi.fd ]] && rm -rf gsi.fd
    $SLINK gsi.fd/src/gsi     gsi.fd
```
In fact, retaining the above erroneously removes the clone of NOAA-EMC/GSI in `gsi.fd`.